src/main/java/organice/logic/parser/ParserUtil.java

```diff
@@ -203,4 +204,19 @@ public static OrganExpiryDate parseOrganExpiryDate(String organExpiryDate) throw
         }
         return new OrganExpiryDate(trimmedOrganExpiryDate);
     }
+
+    /**
+     * Parses a {@code String status} into a {@code Status}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code status} is invalid.
+     */
+    public static Status parseStatus(String status) throws ParseException {
+        requireNonNull(status);
+        String trimmedStatus = status.trim();
+        if (!Status.isValidStatus(trimmedStatus)) {
+            throw new ParseException(Status.MESSAGE_CONSTRAINTS);
+        }
+        return new Status(trimmedStatus);
+    }
```

Comment:
I dont think its necessary tbh, cos the user wont input the `status` directly, rather when u call `processing` command u should call a method call `setProcessing` function that will set the status to under processing

Reply:
make sense. removed